docs/v3_services.rst

```diff
@@ -1,89 +1,105 @@
 SecureDrop V3 Onion Services
 ============================
-Tor onion services provide anonymous inbound connections to websites and other
-servers exclusively over the Tor network. For example, SecureDrop uses onion services
-for the *Journalist* and *Source Interface* websites, as well as for
-adminstrative access to the servers in SSH-over-Tor mode.
+TOr onion services provide anonymous inbound connections to websites and other
+servers exclusively over the Tor network. SecureDrop uses onion services 
+for the *Journalist* and *Source Interface* websites, as well as for 
+adminstrative access to the servers in SSH-over-Tor mode. 
 
-SecureDrop currently supports both the older v2 version of the onion services
-protocol, and the current version, v3. The current version provides stronger
-cryptographic primitives than v2 onion services, and includes redesigned
-protocols that guard against service information leaks on the Tor network.
+Currently, SecureDrop supports two versions of the onion services
+protocol: the older version (v2), and the current version (v3), which provides 
+stronger cryptography and includes redesigned protocols that guard against 
+service information leaks on the Tor network. Because of these important 
+improvements, the Tor project is
+`deprecating support for v2 onion services <https://blog.torproject.org/v2-deprecation-timeline>`__. 
 
-Because of these important improvements, the Tor project is
-`deprecating support for v2 onion services <https://blog.torproject.org/v2-deprecation-timeline>`__.
-SecureDrop will remove support for v2 onion services as part of its 2.0.0
-release, planned for **February 2021**. If you are currently using v2 onion services,
-they will become unreachable at that point.
+SecureDrop will begin phasing out support for v2 onion services in 
+**February 2021**. All newsrooms should enable v3 onion services as soon as
+possible, but may continue using v2 concurrently to allow all journalists and 
+sources to transition smoothly.
 
-The unique identifier in v3 onion addresses is 56 characters long - for example:
+.. warning::
+
+   If you are still using 16-character v2 onion URLs in April 2021, your SecureDrop 
+   servers will become unreachable, and you will have to reinstall SecureDrop 
+   from scratch.
+
+How to distinguish between v2 and v3 onion services
+---------------------------------------------------
+
+If your *Source Interface* address is 56 characters long, you are already using v3 
+onion services:
 
 .. code-block:: none
 
   http://vww6ybal4bd7szmgncyruucpgfkqahzddi37ktceo3ah7ngmcopnpyyd.onion/
 
-This makes them easily distinguishable from 16-character v2 onion addresses,
-such as:
+If your *Source Interface* address is 16-characters long, you are still using 
+v2 onion services, and need to transition to v3 in order to keep using SecureDrop:
 
 .. code-block:: none
 
-  http://secrdrop5wyphb5x.onion/
+  http://secrdrop5wyphb5x.onion/  
```

Comment:
Note that this address is now dead (it was the pre-v2 SecureDrop.org address).

Reply:
It is. But I didn't know what to replace it with, since theoretically all v2s will someday be deprecated (and I didn't want to call out anyone specifically by saying "hey look at _these guys_ still using v2...")  I have left it as is for now, and I think it's clear that the example is just an example.  If strong feelings about this, I will add a ` # just an example of v2 formatting` type comment beside the URL.